core/util.go

```diff
@@ -364,11 +364,13 @@ func GetBuildHost() (retID string) {
 	return
 }
 
-// UniqueNames returns the set of all unique names in the input.
+// UniqueNames returns the set of all unique names in the input after. The
+// returned names will all be lowercased and will have been checked against each
+// other in their lowercase form..
 func UniqueNames(names []string) (unique []string) {
```

Comment:
It's not! There are calls to ToLower in the CA. I was just doing the work doubly. Perhaps, I shouldn't. I'll take a look at the other callers of this.